addons/addon-key-pair-mgmt-api/packages/key-pair-mgmt-services/lib/key-pair/schema/create-key-pair.js

```diff
@@ -22,7 +24,7 @@ const schema = {
       type: 'string',
       maxLength: 100,
       minLength: 2,
-      pattern: '^[A-Za-z0-9-_ ]+$',
+      pattern: nameRegex,
     },
     // Description for this key-pair
     desc: {
```

Comment:
non-block: use `nonHtmlRegex` for desc and test

Reply:
added to CHAM-210 and 211